src/python/dxpy/scripts/dx.py

```diff
@@ -5636,6 +5801,104 @@ def register_parser(parser, subparsers_action=None, categories=('other', ), add_
 parser_publish.set_defaults(func=publish)
 register_parser(parser_publish)
 
+#####################################
+# archive
+#####################################
+                               
+parser_archive = subparsers.add_parser(
+    'archive', 
+    help='Requests for the specified set files or for the files in a single specified folder to be archived on the platform', 
+    description=
+    '''
+Requests for the specified set files or for the files in a single specified folder to be archived on the platform. 
+For each file, if this is the last copy of a file to have archival requested, the full archival of the object will be triggered. 
+Otherwise, the files will be marked in an archival state denoting that archival has been requested.
```

Comment:
```suggestion
Otherwise, the files will be placed in an archival state denoting that archival has been requested.
```

Reply:
The description here is the same as the doc [https://documentation.dnanexus.com/developer/api/data-containers/projects#api-method-project-xxxx-archive](url). Shall I update the documentation as well?